server/adapters/index.js

```diff
@@ -1,5 +1,7 @@
 module.exports = function initAdapters(libs, conf, models, workers) {
   const adapters = {};
 
+  require('./logAdapter')(models, adapters);
```

Comment:
@vuongc 
No no, you continue to add it, like before, to the `adapter` object. But this time instead of adding `logAdapter` to `adapters`, you add `log`. To finally have something like `{ adapters: { log: {...}, foobar: {...} } }`.